pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SinkBase.java

```diff
@@ -85,15 +85,15 @@ public Response registerSink(final @PathParam("tenant") String tenant,
     })
     @Path("/{tenant}/{namespace}/{sinkName}")
     @Consumes(MediaType.MULTIPART_FORM_DATA)
-    public Response updateSink(final @PathParam("tenant") String tenant,
-                               final @PathParam("namespace") String namespace,
-                               final @PathParam("sinkName") String sinkName,
-                               final @FormDataParam("data") InputStream uploadedInputStream,
-                               final @FormDataParam("data") FormDataContentDisposition fileDetail,
-                               final @FormDataParam("url") String functionPkgUrl,
-                               final @FormDataParam("sinkConfig") String sinkConfigJson) {
+    public void updateSink(final @PathParam("tenant") String tenant,
```

Comment:
I don't understand why the return value was changed to `void`?

Reply:
because there is no output that needs to be returned.  If there is an error raising a RestException will trigger jetty to return the error message